src/licenses.ts

```diff
@@ -24,11 +25,40 @@ export async function getInvalidLicenseChanges(
   licenses: {
     allow?: string[]
     deny?: string[]
+    licenseExclusions?: string[]
   }
 ): Promise<InvalidLicenseChanges> {
   const {allow, deny} = licenses
+  const licenseExclusions = licenses.licenseExclusions?.map(
+    (pkgUrl: string) => {
+      return PackageURL.fromString(pkgUrl)
+    }
+  )
 
   const groupedChanges = await groupChanges(changes)
+
+  // filter out changes that are part of exclusions list - config.allow_dependencies_licenses
+  groupedChanges.licensed = groupedChanges.licensed.filter(change => {
+    const changeAsPackageURL = new PackageURL(
+      change.ecosystem,
+      undefined,
+      change.name,
+      change.version,
+      undefined,
+      undefined
+    )
+    if (
+      isDefined(licenseExclusions) &&
+      licenseExclusions.findIndex(
+        exclusion =>
+          exclusion.type === changeAsPackageURL.type &&
+          exclusion.name === changeAsPackageURL.name
+      ) !== -1
+    ) {
+      return false
+    }
+    return true
```

Comment:
This conditional needs some documentation.

Reply:
Added some documentation around the code. Hope it's clear.